README.md

```diff
@@ -1,6 +1,68 @@
+
+# Installation
+* Bleeding edge: `git clone https://github.com/RadioAstronomySoftwareGroup/pyuvsim`
+<!---* pip: `pip install pyuvsim`-->
+* PYUVSim is mainly intended to run on clusters running the linux operating system
+
+## Dependencies
+* `numpy`, `astropy`, `scipy`, `mpi4py`, `pyyaml`, `six`, `pyuvdata`
+* optionally `line_profiler` if you want to do profiling (support for profiling is built in)
+
+# Inputs
+
+A simulation requires sets of times, frequencies, source positions and brightnesses, antenna positions, and direction-dependent primary beam responses. PYUVsim specifies times, frequencies, and array configuration via a UVData object (from the pyuvdata package), source positions and brightnesses via Source objects, and primary beams either through UVBeam or AnalyticBeam objects.
+
+* All sources are treated as point sources, with flux specified in Stokes parameters and position in right ascension / declination in the International Celestial Reference Frame (equivalently, in J2000 epoch).
+* Primary beams are specified as full electric field components, and are interpolated in angle and frequency. This allows for an exact Jones matrix to be constructed for each desired source position.
+* Multiple beam models may be used throughout the array, allowing for more complex instrument responses to be modeled.
+
+These input objects may be made from a data file or from a set of `yaml` configuration files. See [Running a simulation](https://pyuvsim.readthedocs.io/en/latest/usage.html).
+
+# Quick start guide
+Example `obsparam` configuration files may be found in the `reference_simulations` directory.
+
+1. Install from github or pip.
+2. Run off of a parameter file with 20 MPI ranks:
+```
+mpirun -n 20 python run_param_pyuvsim.py -p reference_simulations/obsparam_1.1.yaml
+```
+3. Run with profiling. There is the option to use the `line_profiler` module to assess line-by-line memory and time usage.
+Profiling results are only reported for the rank 0 process if run within MPI.
+```
+mpirun -n 20 kernprof -l -v run_param_pyuvsim.py -p reference_simulations/obsparam_1.1.yaml > profiling_results.out
+```
+
+# Documentation
+Documentation on how to run simulations and developer API documentation is hosted on [ReadTheDocs](https://pyuvsim.readthedocs.io).
+
+# How to contribute
+Contributions to this package to add new features or address any of the
+issues in the [issue log](https://github.com/RadioAstronomySoftwareGroup/pyuvsim/issues) are very welcome.
+Please submit improvements as pull requests against the repo after verifying that
+the existing tests pass and any new code is well covered by unit tests.
+
+Bug reports or feature requests are also very welcome, please add them to the
+issue log after verifying that the issue does not already exist.
+Comments on existing issues are also welcome.
+
+# Versioning Approach
+We use a `generation.major.minor` format.
+* generation - certified comparisons with other simulations
```

Comment:
I would strongly prefer something like "Major algorithmic modifications or substantive new features added.  Always accompanied by in-depth comparisons with other simulations."

Reply:
I'm rethinking this a little bit after talking with Steven and Adam here.   Steven says that when he sees a generation version change he expects a significant change but in major versions new things are "optional". They both agreed that the use of the major version to distinguish "level of validation" while sensible from our perspective, might be excessively novel and perhaps a little opaque for others.  And, on reflection I see that we've under-defined the major category.  We also explored what "algorithmic" meant to us and discovered that to some it implied "physics" and to others "computation". In reality of course they are somewhat correlated. Heres a suggested version where I've stuck to those two words. I've also explicitly enumerated the testing and validation strategy for each version.

Generation - Release combining multiple new physical effects and or major computational improvements. Testing: Backed by unittests, internal model validation,  and significant external comparison.
Major - Adds new physical effect or major computational improvement. Small number of improvements with each release. Testing: Backed by unittests, internal model validation and limited external comparison.
Minor - Bug fixes and small improvements not expected to change physical model. Testing: Backed by Unittests

Reply:
Sounds good to me!
Just to clarify a little for my own understanding... and to give examples of things we've discussed doing:
 - "physical effects"  = things like polarization effects, noise, ionospheric modeling, or nonterrestrial observing positions.
 - "Major computational improvement" = Support for new catalog types (e.g, diffuse maps), new analysis tools, changes to parallelization scheme
 - "Small improvements" = Better documentation or example code, outer framework redesign.

Is this about right?

Reply:
This sounds like a good plan to me.  If you can push the changes, I'll finish the review.